doc/metadata/README.md

```diff
@@ -0,0 +1,20 @@
+Verify in-toto's supply chain with in-toto
```

Comment:
Can we also put some sample links? We can likely sign everything with one key for now and see how verification works.